### DISCUSSION TITLE
[WindowStyle&ResizeMode] Testing

### DISCUSSION THREAD

**Discussion by emako** [Simnico99/MicaWPF#57] [Show and tell]
WindowStyle=None，ResizeMode=NoResize
WindowStyle=None，ResizeMode=CanMinimize
WindowStyle=None，ResizeMode=CanResize
WindowStyle=None，ResizeMode=CanResizeWithGrip
WindowStyle=SingleBorderWindow，ResizeMode=NoResize
WindowStyle=SingleBorderWindow，ResizeMode=CanMinimize
WindowStyle=SingleBorderWindow，ResizeMode=CanResize
WindowStyle=SingleBorderWindow，ResizeMode=CanResizeWithGrip
WindowStyle=ThreeDBorderWindow，ResizeMode=NoResize
WindowStyle=ThreeDBorderWindow，ResizeMode=CanMinimize
WindowStyle=ThreeDBorderWindow，ResizeMode=CanResize
WindowStyle=ThreeDBorderWindow，ResizeMode=CanResizeWithGrip

**Comment by emako** [comment]
> WindowStyle=None，ResizeMode=NoResize

Minimize button: unavailable, invisible

Maximize button: unavailable, invisible

Close button: available

Border resizable: unavailable

---

> WindowStyle=None，ResizeMode=CanMinimize


Minimize button: available, visible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: unavailable

---

> WindowStyle=None，ResizeMode=CanResize

Minimize button: unavailable, invisible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: available

---

> WindowStyle=None，ResizeMode=CanResizeWithGrip

Minimize button: unavailable, invisible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: available

---

> WindowStyle=SingleBorderWindow，ResizeMode=NoResize

Minimize button: unavailable, visible

Maximize button: unavailable, visible

Close button: available, visible

Border resizable: unavailable

---

> WindowStyle=SingleBorderWindow，ResizeMode=CanMinimize

Minimize button: available, visible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: unavailable

---

> WindowStyle=SingleBorderWindow，ResizeMode=CanResize

Minimize button: unavailable, invisible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: available

---

> WindowStyle=SingleBorderWindow，ResizeMode=CanResizeWithGrip

Minimize button: unavailable, invisible

Maximize button: unavailable, invisible

Close button: available, visible

Border resizable: available

---

> WindowStyle=ThreeDBorderWindow，ResizeMode=NoResize

Minimize button: unavailable, visible

Maximize button: unavailable, visible

Close button: available, visible

Border resizable: unavailable

---

> WindowStyle=ThreeDBorderWindow，ResizeMode=CanMinimize

Minimize button: visible, available

Maximize button: invisible, unavailable

Close button: visible, available

Border resizable: unavailable

**Reply by Simnico99** [reply]
Normally the window should work as intended.
